SayTheirNames/Source/Controller/Home/Person/PersonController.swift

```diff
@@ -195,6 +206,18 @@ class PersonController: UIViewController {
 // MARK: - UIView Setup Methods
 private extension PersonController {
     
+    func hideDonationButton() {
+           guard self.person.donations.first != nil else {
+             self.donationButtonContainerView.removeFromSuperview()
+             var viewConstraint: NSLayoutConstraint?
+             viewConstraint = self.tableView.bottomAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.bottomAnchor)
+             viewConstraint?.isActive = true
+               
+             return self.viewBlur.removeFromSuperview()
+           }
+           self.viewBlur.removeFromSuperview()
```

Comment:
I've noticed a few minor issues in this function, the guard is slightly confusing and there's an unnecessary optional var. Also `self.viewBlur.removeFromSuperview()` feels more at home up in the completion. May I suggest to improve the code like this:

```suggestion
        self.viewBlur.removeFromSuperview()
        guard self.person.donations.first == nil else {
            return
        }
        self.donationButtonContainerView.removeFromSuperview()
        let viewConstraint = self.tableView.bottomAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.bottomAnchor)
        viewConstraint.isActive = true
```

Reply:
I want the blur to be removed after the table view constraint has been modified.